rqt_top/msg/CPUUsage.msg

```diff
@@ -0,0 +1 @@
+Process[] processes
```

Comment:
It's certainly something that could be considered standardizing. I'd suggest however to create a separate package with the cpu info, and maybe trying to add some similar info for the status of a computer such as memory info that can be a new package, and be reviewed and merged. The diagnostic messages are very specific to the diagnostic system, which is completely unrelated to cpu info.